### ISSUE TITLE
The accuracy on training dataset

### ISSUE THREAD

**Issue by auroua** [clcarwin/sphereface_pytorch#9]
What's your accuracy on training dataset.
After 20 epochs, I got the following accuracy:
```
17:49:33 Te=19 Loss=2.0825 | AccT=93.3490% (424355/454590) 2.6694 5.00 35525
```
Does this accuracy is  right?  How to improve the accuracy on training dataset?

**Comment by clcarwin** [comment]
Run ```python lfw_eval.py``` to eval your model. If it is not as good as you need, please  fine-tune it.
